### ISSUE TITLE
Failed import - CC.801 from staging.mitx.mit.edu

### ISSUE THREAD

**Issue by carsongee** [mitodl/lore#434]
```
2015-07-29T20:25:37.281653+00:00 app[worker.1]: [2015-07-29 20:25:37,258: ERROR/MainProcess] Task importer.tasks.import_file[2bfc0922-7eb1-4dd3-b441-736ddeba667b] raised unexpected: IOError(u'Error reading file \'/tmp/tmp5oahET/2014_Fall/html/Welcome.html\': failed to load external entity "/tmp/tmp5oahET/2014_Fall/html/Welcome.html"',)
2015-07-29T20:25:37.281656+00:00 app[worker.1]: Traceback (most recent call last):
2015-07-29T20:25:37.281658+00:00 app[worker.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/celery/app/trace.py", line 240, in trace_task
2015-07-29T20:25:37.281659+00:00 app[worker.1]:     R = retval = fun(*args, **kwargs)
2015-07-29T20:25:37.281661+00:00 app[worker.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/celery/app/trace.py", line 438, in __protected_call__
2015-07-29T20:25:37.281662+00:00 app[worker.1]:     return self.run(*args, **kwargs)
2015-07-29T20:25:37.281663+00:00 app[worker.1]:   File "/app/importer/tasks.py", line 13, in import_file
2015-07-29T20:25:37.281665+00:00 app[worker.1]:     import_course_from_file(path, repo_id, user_id)
2015-07-29T20:25:37.281666+00:00 app[worker.1]:   File "/app/importer/api/__init__.py", line 78, in import_course_from_file
2015-07-29T20:25:37.281667+00:00 app[worker.1]:     join(tempdir, path), repo_id, user_id
2015-07-29T20:25:37.281669+00:00 app[worker.1]:   File "/app/importer/api/__init__.py", line 102, in import_course_from_path
2015-07-29T20:25:37.281670+00:00 app[worker.1]:     bundle.import_from_directory(path)
2015-07-29T20:25:37.281672+00:00 app[worker.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/xbundle/__init__.py", line 170, in import_from_directory
2015-07-29T20:25:37.281673+00:00 app[worker.1]:     self.import_course_from_directory(path)
2015-07-29T20:25:37.281675+00:00 app[worker.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/xbundle/__init__.py", line 206, in import_course_from_directory
2015-07-29T20:25:37.281676+00:00 app[worker.1]:     cxml = self.import_xml_removing_descriptor(path, elem)
2015-07-29T20:25:37.281677+00:00 app[worker.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/xbundle/__init__.py", line 370, in import_xml_removing_descriptor
2015-07-29T20:25:37.281679+00:00 app[worker.1]:     dchild = self.import_xml_removing_descriptor(path, child)
2015-07-29T20:25:37.281680+00:00 app[worker.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/xbundle/__init__.py", line 370, in import_xml_removing_descriptor
2015-07-29T20:25:37.281682+00:00 app[worker.1]:     dchild = self.import_xml_removing_descriptor(path, child)
2015-07-29T20:25:37.281683+00:00 app[worker.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/xbundle/__init__.py", line 370, in import_xml_removing_descriptor
2015-07-29T20:25:37.281684+00:00 app[worker.1]:     dchild = self.import_xml_removing_descriptor(path, child)
2015-07-29T20:25:37.281685+00:00 app[worker.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/xbundle/__init__.py", line 345, in import_xml_removing_descriptor
2015-07-29T20:25:37.281687+00:00 app[worker.1]:     join(path, xml.tag, filename), **options).getroot()
2015-07-29T20:25:37.281688+00:00 app[worker.1]:   File "lxml.etree.pyx", line 3310, in lxml.etree.parse (src/lxml/lxml.etree.c:72517)
2015-07-29T20:25:37.281689+00:00 app[worker.1]:   File "parser.pxi", line 1791, in lxml.etree._parseDocument (src/lxml/lxml.etree.c:105979)
2015-07-29T20:25:37.281691+00:00 app[worker.1]:   File "parser.pxi", line 1817, in lxml.etree._parseDocumentFromURL (src/lxml/lxml.etree.c:106278)
2015-07-29T20:25:37.281692+00:00 app[worker.1]:   File "parser.pxi", line 1721, in lxml.etree._parseDocFromFile (src/lxml/lxml.etree.c:105277)
2015-07-29T20:25:37.281693+00:00 app[worker.1]:   File "parser.pxi", line 1122, in lxml.etree._BaseParser._parseDocFromFile (src/lxml/lxml.etree.c:100227)
2015-07-29T20:25:37.281694+00:00 app[worker.1]:   File "parser.pxi", line 580, in lxml.etree._ParserContext._handleParseResultDoc (src/lxml/lxml.etree.c:94350)
2015-07-29T20:25:37.281696+00:00 app[worker.1]:   File "parser.pxi", line 690, in lxml.etree._handleParseResult (src/lxml/lxml.etree.c:95786)
2015-07-29T20:25:37.281697+00:00 app[worker.1]:   File "parser.pxi", line 618, in lxml.etree._raiseParseError (src/lxml/lxml.etree.c:94818)
2015-07-29T20:25:37.281699+00:00 app[worker.1]: IOError: Error reading file '/tmp/tmp5oahET/2014_Fall/html/Welcome.html': failed to load external entity "/tmp/tmp5oahET/2014_Fall/html/Welcome.html"
```

**Comment by tssheth** [comment]
It appears to import fine using the ZIP directly from the github.mit.edu content repo.  However, when the course was loaded via sysadmin dashboard on staging.mitx.mit.edu and then exported via studio, the tar.gz did not import.

**Comment by pdpinch** [comment]
@jetej can you give me a copy of the .tar.gz exported from studio?

The path it's looking for is... odd.. 

At the end of the day, this may point to an issue in XBundle anyway.

**Comment by jetej** [comment]
@pdpinch 
https://www.dropbox.com/s/j6uypkri02hgt8x/2014_Fall.0989j2.tar.gz?dl=0

**Comment by carsongee** [comment]
Did you make that public? Please protect it

**Comment by tssheth** [comment]
Sorry about that. It has been deleted.

**Comment by pdpinch** [comment]
I believe this has been addressed by #766
